### ISSUE TITLE
Custom field logic for VAMC Detail Page to help with accessibility

### ISSUE THREAD

**Issue by cmaeng** [department-of-veterans-affairs/va.gov-cms#7049]
## User Story or Problem Statement

As a CMS editor, I need to be able to see things that will trigger the daily accessibility scan so I can address them before publishing.

CMS helpdesk has gotten multiple requests through #-daily-accessibility-scan to address duplicative heading ids. Many of these are the result of the FE assigning ids to fields on a VAMC detail page where there is no validation or warning for editors to alert them to the accessibility concern.
 
Examples:
- [VAHELP-2249](https://va-gov.atlassian.net/browse/VAHELP-2249?searchSessionId=6cddee6a-5efb-423c-a614-2c416aacd608&searchObjectId=18757&searchContainerId=10009&searchContentType=issue): on [this VAMC detail page
](https://prod.cms.va.gov/central-iowa-health-care/dav-vans-transportation-for-veterans), in the "Related links" section, if the "Title" field and any of the "Link text" fields are identical, the FE will duplicate the heading ids and trip the scan.
- [VAHELP-2625](https://va-gov.atlassian.net/browse/VAHELP-2625): [this VAMC detail page](https://prod.cms.va.gov/providence-health-care/programs/covid-19-vaccines) behaves similarly, where duplication of the "Title" field and any of the "Link text" fields will trip the scan.

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires KB article update

## Implementation steps
The short-term solution may be simply a KB article to guide editors.

Medium-term solution may be inline help text to highlight that the fields need to be unique.

Long-term solution may be to audit all content types where this may come in to play and create custom field logic to validate before an editor is able to publish.

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [x] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [x] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [x] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`
  - [x] `⭐️ Content ops`
  - [x] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product support`
  - [ ] `⭐️ User support`

**Comment by laflannery** [comment]
@EWashb This is the same as the other issue from PW - this duplicate ID thing is not product specific and should be solved overall.  Can we assigned back to CMS? (we might solve it incrementally but I think we need to think about it from a CMS standpoint)

**Comment by laflannery** [comment]
Closing as this is a [duplicate issue](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6603) and also no longer applicable
